.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -53,7 +53,6 @@ jobs:
       env:
         STACKROX_NVD_API_KEY: ${{ secrets.NVD_API_KEY }}
         STACKROX_NVD_API_CALL_INTERVAL: 6s
-      continue-on-error: true
```

Comment:
This was preventing us from being notified about failing steps. We already handle the case when a single matric step fails when others do not via `fail-fast: false`